lib/worker.js

```diff
@@ -80,6 +95,18 @@ Worker.create = function (opts) {
 };
 
 /**
+ * Helper function for reporting errors to rollbar via error-cat. Useful for
+ * testing this functionality.
+ * @param {error} err Error to report.
+ */
```

Comment:
This comment is prompting this in my head: I want to generate some documentation for this repo and have it auto-hosted somewhere (soon). That in mind, it's not actually useful for the _user_ to have "Useful for testing..." in the documentation, so I think I'm going to start avoiding mentioning testing in any jsdocs. :)